systems/plants/@RigidBodyManipulator/contactConstraints.m

```diff
@@ -57,7 +57,8 @@
 % For now, all coefficients of friction are 1
 mu = ones(nC,1);
 
-d = obj.surfaceTangents(normal);
+d = surfaceTangentsmex(normal);
```

Comment:
take a look in collisionDetect.m for an example. 

it might be that checking `obj.mex_model_ptr ~= 0` is sufficient here (since both will happen if eigen exists), but if we go that route then you should comment it to be clear it's for efficiency.

On Jan 31, 2015, at 4:46 PM, John Carter <notifications@github.com> wrote:

> In systems/plants/@RigidBodyManipulator/contactConstraints.m:
> 
> > @@ -57,7 +57,8 @@
> >  % For now, all coefficients of friction are 1
> >  mu = ones(nC,1);
> >  
> > -d = obj.surfaceTangents(normal);
> > +d = surfaceTangentsmex(normal);
> You're right.
> 
> Is there a standard way to check for the presence of Eigen from MATLAB? Would the existence of the compiled mex file be sufficient?
> 
> —
> Reply to this email directly or view it on GitHub.
>